packages/react-components/src/molecules/PeopleCard.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import css from '@emotion/css';
+import format from 'date-fns/format';
+import { UserResponse } from '@asap-hub/model';
+
+import { Card, Link, Headline2, Avatar, Caption } from '../atoms';
+import ProfilePersonalText from './ProfilePersonalText';
+import { tabletScreen } from '../pixels';
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'row',
+  flexWrap: 'wrap',
+  justifyContent: 'space-between',
+  alignItems: 'start',
+});
+
+const textContainerStyles = css({
+  flexGrow: 1,
+  display: 'flex',
+  flexWrap: 'wrap',
+  justifyContent: 'space-between',
+  alignItems: 'start',
+  '> *': {
+    flexBasis: '100%',
+  },
+  [`@media (max-width: ${tabletScreen.min}px)`]: {
+    paddingTop: '12px',
+  },
+});
+
+const moveStyles = css({
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    flexBasis: 'auto',
+    order: -1,
+  },
+});
+
+const avatarContainerStyles = css({
+  paddingRight: '25px',
+});
+
+type PeopleCardProps = Pick<
+  UserResponse,
+  | 'id'
+  | 'avatarURL'
+  | 'department'
+  | 'displayName'
+  | 'firstName'
+  | 'institution'
+  | 'jobTitle'
+  | 'createdDate'
+  | 'lastName'
+  | 'location'
+  | 'teams'
+>;
+const PeopleCard: React.FC<PeopleCardProps> = ({
+  id,
+  department,
+  displayName,
+  institution,
+  createdDate,
+  firstName,
+  lastName,
+  location,
+  teams,
+  jobTitle,
+  avatarURL,
+}) => {
+  return (
+    <Link theme={null} href={`/users/${id}`}>
```

Comment:
I may have missed this in other reviews, but we should pass these hrefs in as props. Route structure is a frontend concern, its definition in `Route`s should not be separated from its usage in `Link`s